### ISSUE TITLE
command 'spacecode.space' not found

### ISSUE THREAD

**Issue by marcoieni** [stevenguh/spacecode#2]
I installed all the required extension, I have a settings file like the one from [VSpaceCode](https://github.com/VSpaceCode/VSpaceCode)

Installed plugins:
![image](https://user-images.githubusercontent.com/11428655/82593675-291d9300-9ba3-11ea-85d8-a32e66396aa5.png)

But when I press space this popup appears:
![image](https://user-images.githubusercontent.com/11428655/82593766-46eaf800-9ba3-11ea-9612-f0a97c65f088.png)

**Comment by stevenguh** [comment]
Did you reload VSCode after your installation?

**Comment by marcoieni** [comment]
yes

**Comment by stevenguh** [comment]
I am sorry that you are encountering issue. I am wondering do you still have the config from `VSpaceCode`? Did you add anything to `spacecode.bindings` in your own `settings.json`?
If you don't mind, can you sharing your `settings.json`?

**Comment by marcoieni** [comment]
This is  exactly my settings.json: https://github.com/VSpaceCode/VSpaceCode/blob/c63380c5fac9690909966a6d53b386cc6a954cde/settings.json

**Comment by marcoieni** [comment]
Anyway, if I completely delete everything from my settings.json I still get the same error.

I am on arch linux and I am using https://github.com/microsoft/vscode

**Comment by stevenguh** [comment]
That's telling me it's not a conflict in `settings.json`. I am thinking somehow the extension's  contribution didn't get registered in vscode. What version of vscode are you using? I don't have linux box I can test right now. But I can try fire up an arch linux vm to try later today.

**Comment by marcoieni** [comment]
```
~ $ code --version
1.45.1
5763d909d5f12fe19f215cbfdd29a91c0fa9208a
x64
```

I don't have the `spacecode.bindings` file, is this required?

**Comment by stevenguh** [comment]
hmm..you have the latest vscode.

Can you go to `defaultSettings.json`(`control+shift+p` and search default setting)? and see if `spacecode.bindings` exists? That should be loaded automatically when you installed the extension.

**Comment by stevenguh** [comment]
Saw the debugging tips from: https://github.com/microsoft/vscode-go/issues/1386

Try open `Help -> Toggle Developer Tools -> Console` and share what you see there when you trying to activate the extension.

**Comment by Mazurel** [comment]
I am also using arch Linux (I don't think it matters) and have the same issue. I have decided to disable the plugin and see what happens when pressing space first time in developer console.

![image](https://user-images.githubusercontent.com/22836301/82731199-6732c800-9d05-11ea-9734-fef00dae4ac0.png)

After a quick look into extension.js and quick ls, I realized that require("./menu/MenuItem") should be changed to require("./menu/menuItem"), aka the imports have a typo. 

(ls for reference)
![image](https://user-images.githubusercontent.com/22836301/82731238-aa8d3680-9d05-11ea-92f8-fbfe08b05337.png)

After changing it, everything works as expected.

**Comment by marcoieni** [comment]
> Can you go to defaultSettings.json(control+shift+p and search default setting)? and see if spacecode.bindings exists? That should be loaded automatically when you installed the extension.

Yes, it is there, I see:
```
	// The bindings of the space key
	"spacecode.bindings": [
		{
			"key": "f",
			"name": "File...",
			"type": "bindings",
			"bindings": [
... and so on
```

> Try open Help -> Toggle Developer Tools -> Console and share what you see there when you trying to activate the extension.


When I try to toggle the extension nothing is showed:
![Peek 2020-05-23 17-36](https://user-images.githubusercontent.com/11428655/82734577-1b3f4d80-9d1c-11ea-88e7-f57760889c72.gif)

By the way I had the same error with Ubuntu 18. The fix by Mazurel makes sense because Windows file system is not case sensitive, so maybe this is why it works on windows

**Comment by stevenguh** [comment]
I released a patch version v0.2.2, can you try it to see if that fixes it?

**Comment by marcoieni** [comment]
Fixed, thanks @Mazurel :+1:
